packages/anon-aadhaar-react/src/components/ProveModal.tsx

```diff
@@ -6,6 +6,7 @@ import { uploadPdf } from '../util'
 import { PasswordInput } from './PasswordInput'
 import { AadhaarPdfValidation } from '../interface'
 import { ErrorToast } from './ErrorToast'
+import { Buffer } from 'buffer'
```

Comment:
I'm wondering why you want to remove the fix. 
Because these are imported on purpose, when running the packages in a web environment it can resolve the 'buffer' dependency.